CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ set(MADNESS_INSTALL_CMAKEDIR "lib/cmake/madness"
 # Build options  ==============================================================
 
 redefaultable_option(MADNESS_BUILD_MADWORLD_ONLY "Build only MADWorld runtime (excluding BLAS/LAPACK interfaces)" OFF)
+redefaultable_option(ENABLE_CEREAL "Use Cereal library along with MADNESS archive for serialization" OFF)
```

Comment:
should rename to make it MADNESS-specific (`MADNESS_ENABLE_CEREAL`)

Reply:
Done with 7412c09.